### ISSUE TITLE
Add ``R`` as an interpreter

### ISSUE THREAD

**Issue by hiphof** [prefix-dev/rattler-build#1249]
Add ``R`` as an interpreter

This should be implemented in a similar way as how ```Perl``` #1229 was implemented.

**In scope**
- Add a ``R`` test section, like is done with ```Perl``` #1230. For how a R-specific test section could look like, see [#790#issuecomment-2509140463](https://github.com/prefix-dev/rattler-build/issues/790#issuecomment-2509140463).
- Add some ``R`` docs (see ``Perl`` example #1234 )

**Comment by wolfv** [comment]
Yes, this would be quite cool! Let me know if you have any questions :)
